packages/create-video/src/init.ts

```diff
@@ -76,10 +93,10 @@ xns(async () => {
 
 	console.log('Get started by running');
 	console.log(chalk.blue(`cd ${selectedDirname}`));
-	console.log(chalk.blue('npm start'));
+	console.log(chalk.blue(shouldUseYarn() ? 'yarn start' : 'npm start'));
 	console.log('');
 	console.log('To render an MP4 video, run');
-	console.log(chalk.blue('npm run build'));
+	console.log(chalk.blue(shouldUseYarn() ? 'yarn build' : 'npm run build'));
```

Comment:
Do we really need to call the function thrice? (L71, L96, L99)
I we can refactor to something like 
```js
const shouldUseYarn = checkYarnUsage();
```